RelayProofsOfConcept/EddsaTokioServer/examples/eddsa_key_gen_client.rs

```diff
@@ -97,7 +76,7 @@ struct EddsaPeer {
     pub message: &'static [u8],
 
     pub agg_key: Option<KeyAgg>,
-    pub R_tot: Option<GE>,
+    pub r_tot: Option<GE>,
```

Comment:
I know it is not standard writing of and you get warnings but I prefer to keep cryptographic parameters the same name as they have in the paper, to keep their meaning (in this case `r` and `R` can be very confusing). To remove the warnings you can use `#![allow(non_snake_case)]`

Reply:
Interesting, Ok, will fix